lib/api/BackbeatAPI.js

```diff
@@ -113,6 +135,114 @@ class BackbeatAPI {
         });
     }
 
+    /**
+     * Get replication backlog in bytes and count
+     * @param {function} cb -callback(error, data)
+     * @return {undefined}
+     */
+    backlog(cb) {
+        async.series([
```

Comment:
It should be fine to query them in parallel, to get that little gain in latency.